app/javascript/redux/greetings/greetings.js

```diff
@@ -0,0 +1,32 @@
+import Axios from 'axios';
+
+// conts
+const FETCH_GREETING = 'hello-rails-react/greetings/FETCH_GREETING';
+
+// actions
+const fetchGreeting = (payload) => ({
+  type: FETCH_GREETING,
+  payload,
+});
+
+// state
+const initialState = [];
+
+export const fetchGreetingApi = () => async (dispatch) => {
+  const returnValue = await Axios.get('api/v1/greetings');
```

Comment:
- [ ] It would be better if we address the comments made by the previous code reviewer. To communicate with the backend, we need to run the server. For example; `localhost:4000/api/v1/greetings` only then can we successful connect the frontend with the backend

Reply:
Hello, @addod19 Thank you for reviewing my project. My app works nicely on both sides have you checked it please with this [ localhost:3000/api/v1/greetings]()
- [x] Chrome
![Screenshot from 2022-08-10 16-24-40](https://user-images.githubusercontent.com/65129228/183912750-f9e13386-02ba-499d-9805-e0f9f6e00d94.png)
- [x] Postman
![Screenshot from 2022-08-10 16-26-38](https://user-images.githubusercontent.com/65129228/183913370-6f05d742-ec4f-4dec-9d0f-37ad190e5434.png)